src/components/partner/partner.service.ts

```diff
@@ -186,6 +193,48 @@ export class PartnerService {
     return await mapListResults(results, (dto) => this.secure(dto, session));
   }
 
+  async listProjects(
+    partner: Partner,
+    input: ProjectListInput,
+    session: Session
+  ): Promise<SecuredProjectList> {
```

Comment:
Let's forward to ProjectService.list here with a new partner id filter. This will allow us to load everything in one query and keep sorting & pagination working.

[Here's](https://github.com/SeedCompany/cord-api-v3/blob/66c8ee3447b8593f3f21ef4cf0f7cf97e245c0eb/src/components/project/project.service.ts#L453-L483) an example for listing ProjectMembers of a Project.

Reply:
I modeled this after what is happening on the Language Detail page.  I'm unclear on what you mean here.  Do you mean that instead of even adding/calling this new listProjects method in the PartnerService I should instead call ProjectService.list?  Or that from within this PartnerService.listProjects I should call ProjectService.list?

Reply:
The latter. The example I linked above does the latter as well.